website/source/docs/platform/k8s/run.html.md

```diff
@@ -67,7 +67,25 @@ $ helm install --name vault ./
 ...
 ```
 
-_That's it._ The Helm chart does everything to setup a Vault-on-Kubernetes deployment.
+!> **IMPORTANT NOTE:** Vault Helm will not initialize and unseal Vault out of the box.  
+Initialization is required after installation followed by an unseal.  Vault can be 
+configured to auto-unseal using KMS providers such as 
+[Google Cloud Platform](/docs/platform/k8s/run.html#google-kms-auto-unseal).  This 
+allows the pods to auto-unseal if they're rescheduled in Kubernetes.
+
+If standalone or HA mode are being used, the Vault pods must be initialized and unsealed.  
+For HA deployments, only one of the Vault pods needs to be initialized.
+
+```sh
+$ kubectl exec -ti vault-0 -- vault operator init
```

Comment:
Question: does this need to be done on the leader? Is `vault-0` assumed to be the leader here?

Reply:
`vault-0` will always be there since it's the first ordinal pod.  It just makes things easier to make the first pod the leader but isn't required.

Reply:
I suppose a better question is "is there a leader before `init` is ran?"